src/main/resources/fxml/terrain_exaggeration.fxml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<?import javafx.scene.control.Label?>
+<?import javafx.scene.control.Slider?>
+<?import javafx.scene.layout.StackPane?>
+<?import javafx.scene.layout.VBox?>
+
+<?import com.esri.arcgisruntime.mapping.view.SceneView?>
+
+<StackPane fx:controller="com.esri.samples.scene.terrain_exaggeration.TerrainExaggerationController" xmlns:fx="http://javafx.com/fxml"
+           stylesheets="/css/style.css">
+    <!--SDK SceneView-->
+    <SceneView fx:id="sceneView"/>
+    <!--Slider box-->
+    <VBox StackPane.alignment="TOP_LEFT" maxWidth="200" maxHeight="50" spacing="5" styleClass="panel-region">
+        <Label text="Exaggeration"/>
+        <Slider fx:id="exaggerationSlider" max="5" majorTickUnit="1" minorTickCount="0" showTickLabels="true" 

```

Comment:
add `showTickMarks = true`. This adds to the intuition that the slider should snap to the labels.